server/routes/createPlan/typeOfWorkExperience/typeOfWorkExperienceController.test.ts

```diff
@@ -116,6 +121,8 @@ describe('TypeOfWorkExperienceController', () => {
       res.redirect.mockReset()
       next.mockReset()
       validationMock.mockReset()
+      mockCiagService.updateCiagPlan.mockReset()
+      mockInductionService.updateInduction.mockReset()
```

Comment:
Good shout. I'll make sure I'm doing this in all my corresponding tests.

Reply:
I think I copied this from your examples anyway! 🤣